### PR TITLE
moosequery - Addition of targetThrough sourceThrough throughAllFrom throughAllTo

### PR DESCRIPTION
**Pull request by LionelAkue**
Enhance queryLocal:For:in:

### CODE REVIEW

**Comment by anneetien** [comment]
These commits aim to add throughFrom:aFxAssociation query / throughTo: aFxAssociation and all the associated methods.
These methods respectively return the source / target of all Associations pointing to / starting from self.
self throughFrom: FamixInvocation equivalent to : (self queryIncoming: FamixInvocation) collect: #from

**Comment by LionelAkue** [comment]
Hello

I made a commit fixing some comments and adding some rationale.

Thanks

Lionel

**Comment by LionelAkue** [comment]
Tests broken.
Work needed

**Comment by LionelAkue** [comment]
Related tests are green

File: src/Moose-Query/TOODependencyQueries.trait.st [hunk 1]
```diff
@@ -31,3 +31,15 @@ TOODependencyQueries classSide >> generatedTraitNames [
 	<generated>
 	^ #(TDependencyQueries)
 ]
+
+{ #category : #'moosequery-queries-generic' }
+TOODependencyQueries >> throughAllFrom [
+	"Query all the incoming associations of the receiver and its children.
+
+	Example:
+		aFAMIXClass queryAllIncoming.
```
Comment (jecisc) [inline src/Moose-Query/TOODependencyQueries.trait.st]: Are you sure of the comments?

Comment (jecisc) [reply src/Moose-Query/TOODependencyQueries.trait.st]: ping @LionelAkue

**Pull request merged**